internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -739,6 +740,12 @@ resource "azurerm_mysql_flexible_server" "test" {
 
   sku_name = "GP_Standard_D2ds_v4"
   zone     = "1"
+
+  lifecycle {
+    ignore_changes = [
+      high_availability.0.standby_availability_zone
```

Comment:
could you explain why optional + computed is not a better solution here?

Reply:
@katbyte thanks for your quick review. Per the comment [here](https://github.com/hashicorp/terraform-provider-azurerm/blob/6293b0379ffe39c6f46e594c06103112f5214595/vendor/github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema/zone.go#L8), I assume that the optional + computed is not expected. Please correct me if I understand wrong.